declarations/DllModuleFactory.d.ts

```diff
@@ -0,0 +1,27 @@
+export = DllModuleFactory;
+declare const DllModuleFactory_base: typeof import("./ModuleFactory");
+/** @typedef {import("./ModuleFactory").ModuleFactoryCreateData} ModuleFactoryCreateData */
```

Comment:
These jsdoc comments seem to be wrong here.

Looks like they are incorrectly copied when they are directly before the class declaration.

Typescript issue